### PR TITLE
Add desktop horizontal scrolling and mobile indication

### PR DESCRIPTION
**Pull request by codebase-admin**
Does two main things! First it adds a new horizontal scrolling component in desktop mode (anything >= 768 px in width) so that as the user scrolls down vertically, the section scrolls horizontally. This doesn't work too well in mobile mode, so added a blinking indication telling user to swipe right.

### CODE REVIEW

File: src/components/home/home-experiences.js [hunk 1]
```diff
@@ -55,30 +56,43 @@ const HomeExperiences = () => {
...
+            <p>A glimpse into our community</p>
+            {width < 768 && (
+              <p className="cb-home-experiences-swipe">
+                Swipe or scroll to explore →
+              </p>
+            )}
+          </div>
+        </div>
+      </Col>
+      {width > 480 ? (
```
Comment (rrrliu) [inline src/components/home/home-experiences.js:78]: Should we change this to 768 to make sure the mobile and desktop views use the same cutoff? Also consider saving as a constant -> `const cutoff = 768`.

File: src/styles/home-experiences.css [hunk 2]
```diff
@@ -60,23 +42,83 @@
...
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
 @media only screen and (max-width: 575px) {
```
Comment (rrrliu) [inline src/styles/home-experiences.css:76]: Same here – should we put this under the 767px @media rule?

Comment (codebase-admin) [reply src/styles/home-experiences.css]: u right :))

File: src/components/home/home-experiences.js (hunk 1 above)
Comment (codebase-admin) [reply src/components/home/home-experiences.js]: yes def

**Pull request merged**